test/thrift/binary/framed/ssl_test.exs

```diff
@@ -0,0 +1,62 @@
+defmodule Servers.Binary.Framed.SSLTest do
+  use ThriftTestCase
+
+  @thrift_file name: "ssl_test.thrift", contents: """
+    service SSLTest {
+      bool ping();
+    }
+  """
+
+  alias Servers.Binary.Framed.SSLTest.SSLTest.Binary.Framed.{Client, Server}
+
+  def define_handler do
+    defmodule SSLTestHandler do
+      alias Servers.Binary.Framed.SSLTest.SSLTest.Handler
+      @behaviour Handler
+      def ping, do: true
+    end
+  end
+
+  setup_all do
+    {:module, mod_name, _, _} = define_handler()
+    {:ok, _} = Server.start_link(mod_name, 0, [ssl_opts: [enabled: true, configure: &get_certs/0]])
+    server_port = :ranch.get_port(mod_name)
+
+    {:ok, handler_name: mod_name, port: server_port}
+  end
+
+  setup(ctx) do
+    {:ok, client} = Client.start_link("localhost", ctx.port, ssl_opts: ctx[:ssl_opts] ++ [enabled: true])
+
+    {:ok, client: client}
+  end
+
+  @tag ssl_opts: []
+  thrift_test "it can return a simple boolean value", ctx do
+    assert {:ok, true} == Client.ping(ctx.client)
+  end
+
+  @tag ssl_opts: [configure: {__MODULE__, :test_configure, []}]
+  thrift_test "it can handle live configuration", ctx do
+    assert {:ok, true} == Client.ping(ctx.client)
+    assert_received :configured
+  end
+
+  ## Helpers
+
+  defp get_certs() do
+    certs_path = Application.app_dir(:ssl, "examples/certs/etc/server")
```

Comment:
Are these files going to be added to the repo?

Reply:
These files are shipped with OTP for testing, they are inside the `:ssl` application. We can put them inside elixir-thrift if you like.

Reply:
Ah, I missed the part where it refers to the`:ssl` app. I like it the way you have it now.